LFPy/test/test_eegmegcalc.py

```diff
@@ -185,6 +185,44 @@ def test_rad_sign(self):
         s_vector = fs._sign_rad_dipole(P1)
         np.testing.assert_almost_equal(s_vector, np.array([1., -1.]))
 
+    def test_calc_vn(self):
+        print('test calc_vn')
+        n = 1
+        fs = make_simple_class_object()
+        v1 = fs._calc_vn(1)
+        np.testing.assert_almost_equal(v1, -4.75)
+
+    def test_calc_yn(self):
+        print('test calc_vn')
+        n = 1
+        fs = make_simple_class_object()
+        y1 = fs._calc_yn(1)
+        np.testing.assert_almost_equal(y1, -2.3875)
+
+    def test_calc_zn(self):
+        print('test calc_vn')
+        n = 1
+        fs = make_simple_class_object()
+        z1 = fs._calc_zn(1)
+        np.testing.assert_almost_equal(z1, -2.16574585635359)
+
+    def test_calc_potential(self):
```

Comment:
This test failed locally. Can you please investigate a bit?

Reply:
Hi @solveignaess 
Just a summary of what I did. I had to revert your commit 76505ce as it removed the whole cell.py file ;-)
Otherwise I merged master back in your working branch and pushed the changes. Looks good, but there is now this one new test that fails (tested locally).